base/ltfinal.dtx

```diff
@@ -1325,7 +1325,25 @@
 %    \end{macrocode}
 %  \end{macro}
 %
+%   To allow package and class author to support for document links
+%   we provide also the new interface commands of the hyperref package
+%   for the creation of targets.
 %
+%  \begin{macro}{\MakeLinkTarget}
+%  \begin{macro}{\LinkTargetOn}
+%  \begin{macro}{\LinkTargetOff}
+%  \begin{macro}{\NextLinkTarget}
```

Comment:
there should probably be a rollback/rollforward as discussed.

Reply:
i will try to figure it out ;-)

Reply:
I can do that for you if you like

Reply:
I guess that remark is obsolete (I thought we introduced \DocumentMetadata already in the previous release but since that is not the case it rolls back with the new module)

Reply:
Really? the hyperref commands are in ltfinal and not in ltmeta. (I could move them there if you think the location is better. I used ltfinal only because the IfPDFManagement-test was there).

Reply:
ah right, but they belong there don't they it is part of the ltmeta machinery by the end of the day.

Reply:
ok I moved them to ltmeta and reverted the change in ltfinal.